hm_cfg_files/config/CFG/radioss2024/LOADS/detpointset.cfg

```diff
@@ -0,0 +1,43 @@
+//Copyright>    CFG Files and Library ("CFG")
+//Copyright>    Copyright (C) 1986-2023 Altair Engineering Inc.

```

Comment:
@Schwoertzig 

please change 2023 -> 2024

Reply:
Thank you @sebastienVilleneuve for your feedback.
It is now updated.
Regards